lib/main.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'game'
+require_relative 'board'
+require_relative 'player'
+require_relative 'game_cycle'
+
+board = Board.new
+player1 = Player.new('Player_1', x = nil,board )
+player2 = Player.new('Player_2',x = player1.token,board)
+game = Game.new(player1,player2,board)
+game.switch_player
+board.populate_display
```

Comment:
One last thing. I would expect that since you are already calling game_cycle in this file (thru the include Game_cycle line below) you won't need to put game.switch_player and board.populate_display here anymore. In essence in your Game_cycle.game_cycle method should already be the one making sure the game is continuing until there is a winner or if it's a tie. 
Putting game.switch_player and board.populate_display in this file might be making some of the method calls inside game_cycle redundant. For instance, in game_cycle you are already calling game.switch_player and also game.play_turn (play_turn in turn already does the board.populate_display method for you).